### DISCUSSION TITLE
How to cleanup site list?

### DISCUSSION THREAD

**Discussion by yinheli** [darkreader/darkreader#8967] [Show and tell]
After months usage, my site list get mess.

![image](https://user-images.githubusercontent.com/235094/170690972-866d1e39-c116-4b7e-88b2-a9b4684594cf.png)

How can i clean the list once, instead of remove of them one by one?

Use dev tool!

![image](https://user-images.githubusercontent.com/235094/170691258-61b375fd-9cf5-47bd-9c42-d02b0c978e31.png)

execute following in console

```js
var s = null
chrome.storage.sync.get(null, (obj) =>s=obj)

setTimeout(()=>{
  s.siteList = []
  s.siteListEnabled= []
  chrome.storage.sync.set(s, ()=>{})
}, 2000)
```

> caution!!! that will CLEAN all you site list!

**Comment by Gusted** [comment]
A one-liner:

```js
chrome.storage.sync.get(null, (settings) => chrome.storage.sync.set({...settings, siteList: [], siteListEnabled: []}, void 0)); 
```
